### PR TITLE
Bonnie++ foss2016a

### PR DESCRIPTION
**Pull request by RvDijk**
Without the DB eb files.

### CODE REVIEW

**Comment by boegel** [comment]
@RvDijk The easyconfig for DB is still there... You can check the diff when opening a PR.

I don't mind merging both together if that makes it easier on you.

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
@RvDijk Or, you can submit a separate PR with only the DB easyconfig file (commit e65dfd7)

**Comment by RvDijk** [comment]
@boegel strange, I thought it would be removed. I'm working on removing the DB eb file.

**Comment by RvDijk** [comment]
@boegel should be fixed!

File: easybuild/easyconfigs/b/Bonnie++/Bonnie++-1.97-foss-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+sanity_check_paths = {
+    'files': ['sbin/bonnie++'],
+    'dirs': []
+}
+
+moduleclass = 'tools'
+
```
Comment (boegel) [inline easybuild/easyconfigs/b/Bonnie++/Bonnie++-1.97-foss-2016a.eb]: style nitpicking: please drop the trailing empty line

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2135.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/af88dd8c870857ddb10e48099c148602 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2653.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/80b2cde4715e346ca6dc2bcba2871e69 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
@RvDijk good to go, but I'd like to get the tiny style remark fixed first, shouldn't be hard ;-)

**Comment by boegel** [comment]
Going in, thanks @RvDijk!

**Pull request merged**